x-pack/plugins/alerting/common/rules_settings.ts

```diff
@@ -44,8 +44,13 @@ export const RULES_SETTINGS_SAVED_OBJECT_ID = 'rules-settings';
 export const DEFAULT_LOOK_BACK_WINDOW = 20;
 export const DEFAULT_STATUS_CHANGE_THRESHOLD = 4;
 
-export const DEFAULT_FLAPPING_SETTINGS = {
+export const DEFAULT_FLAPPING_SETTINGS: RulesSettingsFlappingProperties = {
   enabled: true,
   lookBackWindow: 20,
```

Comment:
Not related to this PR, but does it make sense to use `DEFAULT_LOOK_BACK_WINDOW` and `DEFAULT_STATUS_CHANGE_THRESHOLD` in the `DEFAULT_FLAPPING_SETTINGS`? Like:

```suggestion
  lookBackWindow: DEFAULT_LOOK_BACK_WINDOW,
```

Reply:
Yes, I think that makes sense I can update that